src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-content: center;
+  align-items: center;
+
+  &__star {
+    display: flex;
+    place-content: center;
```

Comment:
The 'place-content' shorthand property is not necessary here if you're already using 'align-items' to center the content vertically. If you're not using 'justify-content', you can remove this property.